hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/Table.java

```diff
@@ -271,6 +271,15 @@ List<? extends KeyValue<KEY, VALUE>> getSequentialRangeKVs(KEY startKey,
           MetadataKeyFilters.MetadataKeyFilter... filters)
           throws IOException, IllegalArgumentException;
 
+  /**
+   * Delete all keys with a prefix.
```

Comment:
Deletes all keys with the specified prefix from the metadata store as part of a batch operation.

Reply:
Thanks, the new comment is better.